### PR TITLE
[de] Fix typos in `README`

### PR DESCRIPTION
**Pull request by delusko22**
I fixed some crucial typos and errors in the README-de.md!

### CODE REVIEW

File: README-de.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+Sobald Ihre Pull-Request erstellt wurde, übernimmt ein Rezensent von Kubernetes die Verantwortung für klares, umsetzbares Feedback. Als Eigentümer des Pull-Request **liegt es in Ihrer Verantwortung Ihre Pull-Reuqest entsprechend des Feedbacks, welches Sie vom Kubernetes-Reviewer erhalten haben, abzuändern.** Beachten Sie auch, dass Sie am Ende mehr als einen Rezensenten von Kubernetes erhalten, der Ihnen Feedback gibt, oder dass Sie Rückmeldungen von einem Rezensenten von Kubernetes erhalten, der sich von demjenigen unterscheidet, der ursprünglich für das Feedback zugewiesen wurde. In einigen Fällen kann es vorkommen, dass einer Ihrer Prüfer bei Bedarf eine technische Überprüfung von einem [Kubernetes Tech-Reviewer](https://github.com/kubernetes/website/wiki/tech-reviewers) anfordert. Reviewer geben ihr Bestes, um zeitnah Feedback zu geben, die Antwortzeiten können jedoch je nach den Umständen variieren.
```
Comment (kbhawkey) [inline README-de.md:12]: nit:  Pull-Reuqest ?

Comment (delusko22) [reply README-de.md:12]: Even though "Pull-Anfrage" might be correctly translated, "Pull-Request", the English version, is commonly used in German.

Comment (kbhawkey) [reply README-de.md:12]: OK, but should it be:
Pull-Request not `Pull-Reuqest`?

Comment (delusko22) [reply README-de.md]: Yes, I corrected it!

**Comment by kbhawkey** [comment]
Ping `de` localization team for review
/cc @kubernetes/sig-docs-de-reviews

**Comment by mehabhalodiya** [comment]
I'm afraid I won't be able to help with reviewing PRs on the `German` language side as I can't read `German`.

**Pull request merged**